ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/ui/landing/LandingScreen.kt

```diff
@@ -56,18 +62,21 @@ import com.example.android.wearable.composeadvanced.presentation.ui.util.ReportF
  */
 @Composable
 fun LandingScreen(
+    scalingLazyListState: ScalingLazyListState,
+    focusRequester: FocusRequester,
     onClickWatchList: () -> Unit,
     onClickDemoUserInputComponents: () -> Unit,
     onClickDemoMap: () -> Unit,
     onClickDialogs: () -> Unit,
     proceedingTimeTextEnabled: Boolean,
     onClickProceedingTimeText: (Boolean) -> Unit,
```

Comment:
Consider adding a modifier to all composable functions

Reply:
https://github.com/android/wear-os-samples/pull/351